gradle.properties

```diff
@@ -12,4 +12,5 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemor
 org.gradle.parallel=true
 
 # Necessary to turn off certain R8 optimizations that would otherwise cause Shattered to crash
-android.enableR8.fullMode=false
\ No newline at end of file
+android.enableR8.fullMode=false
```

Comment:
check the comment above. are you sure it's not a problem?

Reply:
Hello, you can ignore this commit. I personally prefer to disable minify and obfuscation of build files during compilation, so that game applications will be more friendly for developers to debug.